archive/compression/compression_test.go

```diff
@@ -18,11 +18,40 @@ package compression
 
 import (
 	"bytes"
+	"compress/gzip"
+	"crypto/md5"
+	"io"
 	"io/ioutil"
 	"math/rand"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
 	"testing"
 )
 
+const benchmarkTestDataURL = "https://git.io/fADcl"
```

Comment:
I don't think its a great idea to call out to an external url every time the tests are run.

Reply:
```
Can we move test data to gist or somewhere else? (could be downloaded from TestMain())
```

@AkihiroSuda @stevvooe could you please decide which approach works for you?

Reply:
@dmcgowan oh, ok. I've moved the benchmark to https://git.io/fASMy and added `sync.Once` for initialization